src/Helper/Helper_Abstract_Pdf_Shortcode.php

```diff
@@ -164,6 +165,10 @@ protected function get_pdf_config( $entry_id, $pdf_id ) {
 	public function gravitypdf_confirmation( $confirmation, $form, $entry ) {
 
 		/* check if confirmation is text-based */
+		if ( is_wp_error( $confirmation ) ) {
```

Comment:
This should go above the comment on line 167. There should also be an inline comment explaining why we are checking for a WP Error with a link back to GitHub. This applies to all uses in this file.